### PR TITLE
[Draft] [onert] Support Q16 SYMM model on TRIX Backend

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit supports Q16 symmetric model on TRIX Backend.
It defines new TYPE:
  nnfw.h: NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED
  DataType.h: QUANT_INT16_SYMM

Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Comment by jyoungyun** [comment]
DATA_TYPE_QSYMM16 is a signed int16 symmetric quantization.

[related code (need to login)](https://review.tizen.org/gerrit/gitweb?p=platform/adaptation/npu/trix-engine.git;a=blob;f=src/core/ne-data.cc;h=f46c6f2118103f929209f3c691f22405cd3e488f;hb=refs/heads/tizen#l27)

**Comment by hseok-oh** [comment]
LGTM

**Review by chunseoklee — APPROVED** [review]
LGTM

File: tests/tools/nnpackage_run/src/h5formatter.cc [hunk 1]
```diff
@@ -143,6 +143,8 @@ void H5Formatter::loadInputs(const std::string &filename, std::vector<Allocation
           else
             throw std::runtime_error("model input type is int8. But h5 data type is different.");
           break;
+        case NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED:
+          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/h5formatter.cc:147]:
```suggestion
          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
          break;
```

File: tests/tools/nnpackage_run/src/h5formatter.cc [hunk 2]
```diff
@@ -234,6 +236,8 @@ void H5Formatter::dumpOutputs(const std::string &filename, std::vector<Allocatio
           data_set.write(outputs[i].data(), H5::PredType::NATIVE_INT8);
           break;
         }
+        case NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED:
+          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/h5formatter.cc:240]:
```suggestion
          throw std::runtime_error("NYI for NNFW_TYPE_TENSOR_QUANT16_SYMM_SIGNED type");
          break;
```

**Review by glistening — APPROVED** [review]
LGTM

File: tests/tools/nnpackage_run/src/h5formatter.cc (hunk 1 above)
Comment (hseok-oh) [reply tests/tools/nnpackage_run/src/h5formatter.cc:147]: I've introduced similar code (https://github.com/Samsung/ONE/pull/8883/files) but it doesn't make new warning, fortunately. (analysis tool version update?)

**Comment by hseok-oh** [comment]
@jyoungyun Can't we close this draft?

**Comment by hseok-oh** [comment]
All merged

**Pull request closed without merging**